src/cache/diagnostic_edmf_precomputed_quantities.jl

```diff
@@ -336,13 +336,15 @@ NVTX.@annotate function set_diagnostic_edmf_precomputed_quantities_do_integral!(
         Fields.field_values(Fields.level(Fields.coordinate_field(Y.f).z, half))
 
     # integral
+    @. p.scratch.ᶜtemp_scalar_3 = Y.c.ρq_tot / Y.c.ρ
```

Comment:
By the way, you can improve readability by defining
```
q_tot = p.scratch.ᶜtemp_scalar_3
```
so that expressions below will have `q_tot` instead of `p.scratch.ᶜtemp_scalar_3`